From e2bafe8e7419e034901e80f36113d93147b02a2d Mon Sep 17 00:00:00 2001
From: Lauri Tirkkonen <lauri@hacktheplanet.fi>
Date: Wed, 23 Jul 2025 10:27:43 +0900
Subject: [PATCH] blkdiscard: ship link to /sbin instead of /usr/bin

util-linux ships blkdiscard in /sbin, so follow what it does.

Link: https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/tree/sys-utils/Makemodule.am?id=5a75fb2b9e384c61a4b2afd13fe9bba219d5bd93#n194
Signed-off-by: Lauri Tirkkonen <lauri@hacktheplanet.fi>
---
 util-linux/blkdiscard.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util-linux/blkdiscard.c b/util-linux/blkdiscard.c
index 70fd34af3..cef25c896 100644
--- a/util-linux/blkdiscard.c
+++ b/util-linux/blkdiscard.c
@@ -11,7 +11,7 @@
 //config:	help
 //config:	blkdiscard discards sectors on a given device.
 
-//applet:IF_BLKDISCARD(APPLET_NOEXEC(blkdiscard, blkdiscard, BB_DIR_USR_BIN, BB_SUID_DROP, blkdiscard))
+//applet:IF_BLKDISCARD(APPLET_NOEXEC(blkdiscard, blkdiscard, BB_DIR_SBIN, BB_SUID_DROP, blkdiscard))
 
 //kbuild:lib-$(CONFIG_BLKDISCARD) += blkdiscard.o
 
-- 
2.50.1

